.github/workflows/tests.yml

```diff
@@ -14,5 +14,5 @@ jobs:
       - uses: actions/checkout@v1
       - name: Run Tests
         run: |
-          npm install 
+          npm ci --omit=optional 
```

Comment:
oh, I didn't know about this command. cool!

Reply:
> I'm very glad you're able to add storybook into our codebase. I and maybe others don't have too much experience with storybook. Could you write a short blurb on what is storybook and how to onboard onto it, perhaps here: https://github.com/waterthetrees/waterthetrees/wiki/Onboarding ?

Sure, I'll get on that tomorrow.

> My biggest concern is how this will affect the development process for those that haven't onboarded onto storybook? Will it still be possible to make front-end changes without making a story for the target UI component? Approving this with the assumption that this is the case from what I've read on https://storybook.js.org/. Are there any other helpful links to learn more?

Storybook is optional but recommended for UI development. It's not required to create a story for a UI component, but you may find that having a story makes development easier. I'll try to fill out the onboarding with links.

> Another question is how this will change our tests? It seems like we should still write tests in jest / react testing library, is that correct? Or do we abandon jest since storybook comes with its own test runner? Are there any paradigm changes necessary here?

You can continue to make unit tests in Jest, and you can even pull your stories into Jest to prevent code duplication of mocks and so on (as you can see with Tree.test.js). It's true that you could write unit tests in Storybook ([here](https://www.youtube.com/watch?v=wEa6W8uUGSA)'s a video that goes over it), but I think for consistency let's just keep using Jest.